TCL4 Data Management/utm-tcl4-dmp-con.yaml

```diff
@@ -181,10 +178,6 @@ definitions:
     required:
       - "metaData"
       - "timestamp"
-      - "temperature"
-      - "pressure"
-      - "windSpeed"
```

Comment:
these should be mandatory otherwise you can have an object with zero data

Reply:
The problem is that not every sensor will have all those measurements.

Reply:
ok